### ISSUE TITLE
(M2C)  check_existence timestamps to be updated (blocked by / simult w   #284)

### ISSUE THREAD

**Issue by ndushay** [sul-dlss/preservation_catalog#286]
PC timestamps:
if check_existence clean:
- [ ] last_checked_on_storage AND 
- [ ] last_audited AND 
- [ ] updated_at (but ActiveRecord does this, right?)

PO timestamp(s) changes only when check_existence:
- calls create_with_validation (already done in plain create method, timestamp info in ticket #288)
- calls update_version_with_validation (timestamp info already in ticket #287)

See #284

**Comment by ndushay** [comment]
It is believed that these timestamps will be updated in confirm_version happy path;  when confirm_version is NOT in happy path ... should timestamps be updated by confirm_version?

When check_existence calls confirm version or update_version_after_vdn or create_after_vdn, these timestamp updates will be done within those methods, no?

So again, it's the same question for the UNHAPPY path for both confirm_version and check_existence -- do we update the timestamps?

**Comment by ndushay** [comment]
Believe this was addressed with PR for #284  cloing
